### PR TITLE
Swapped pendant in favor or earrings

### PR DESCRIPTION
**Pull request by adamgoerdt**
Fixes issue 1757

### CODE REVIEW

File: crossing-training.lic [hunk 1]
```diff
@@ -1009,7 +1009,7 @@ class CrossingTraining
   def shape(chapter, unique_name, item)
     return unless buy_wood
 
-    find_shaping_room(@hometown, @engineering_room)
```
Comment (rpherbig) [inline crossing-training.lic]: Is this supposed to change?

**Comment by rpherbig** [comment]
I added the recipe and closed #1757. Do you still want to make this change?

**Comment by adamgoerdt** [comment]
If the recipe fixes it then we don't need to swap the product.  I'll cancel
this PR

**Comment by Sarvatt** [comment]
It's not buying the cord needed for the pendant and still getting hung up.

**Comment by rpherbig** [comment]
Is this after restarting dependency?

**Comment by Sarvatt** [comment]
Yep restarted the client and I had wood pendant in my base-recipes when the error hit again. The cords are in a crate and not bought via ORDER, not sure if that matters.

**Comment by rpherbig** [comment]
I don't know if the script supports that. Good call.

**Pull request closed without merging**